lib/GuestManager.php

```diff
@@ -69,6 +76,125 @@ public function updateName(Room $room, Participant $participant, string $display
 		}
 	}
 
+	public function validateMailAddress(string $email): bool {
+		return $this->mailer->validateMailAddress($email);
+	}
+
+	/**
+	 * @return array{invites: non-negative-int, duplicates: non-negative-int, invalid?: non-negative-int, invalidLines?: list<non-negative-int>, type?: int<-1, 6>}
+	 * @throws GuestImportException
+	 */
+	public function importEmails(Room $room, $file, bool $testRun): array {
+		if ($room->getType() === Room::TYPE_ONE_TO_ONE
+			|| $room->getType() === Room::TYPE_ONE_TO_ONE_FORMER
+			|| $room->getType() === Room::TYPE_NOTE_TO_SELF
+			|| $room->getObjectType() === BreakoutRoom::PARENT_OBJECT_TYPE
+			|| $room->getObjectType() === Room::OBJECT_TYPE_VIDEO_VERIFICATION) {
+			throw new GuestImportException(GuestImportException::REASON_ROOM);
+		}
+
+		$content = fopen($file['tmp_name'], 'rb');
+		$details = fgetcsv($content, escape: '');
+		if (!isset($details[0]) || strtolower($details[0]) !== 'email') {
+			throw new GuestImportException(
+				GuestImportException::REASON_HEADER_EMAIL,
+				$this->l->t('Missing email field in header line'),
+			);
+		}
+		if (isset($details[1]) && strtolower($details[1]) !== 'name') {
+			throw new GuestImportException(
+				GuestImportException::REASON_HEADER_NAME,
+				$this->l->t('Missing name field in header line'),
+			);
+		}
+
+		$participants = $this->participantService->getParticipantsByActorType($room, Attendee::ACTOR_EMAILS);
+		$alreadyInvitedEmails = array_flip(array_map(static fn (Participant $participant): string => $participant->getAttendee()->getInvitedCloudId(), $participants));
+
+		$line = $duplicates = 0;
+		$emailsToAdd = $invalidLines = [];
+		while ($details = fgetcsv($content, escape: '')) {
+			$line++;
+			if (isset($alreadyInvitedEmails[$details[0]])) {
+				$this->logger->debug('Skipping import of ' . $details[0] . ' (line: ' . $line . ') as they are already invited');
+				$duplicates++;
+				continue;
+			}
+
+			if (count($details) > 2) {
+				$this->logger->debug('Invalid entry with too many fields on line: ' . $line);
+				$invalidLines[] = $line;
+				continue;
+			}
+
+			if (count($details) === 2) {
+				[$email, $name] = $details;
```

Comment:
```suggestion
				[$email, $name] = $details;
```

Would it be wise to add a `trim()` to these values?